core-rust/core-api-server/Cargo.toml

```diff
@@ -23,13 +23,13 @@ parking_lot = { version = "0.12" }
 #
 # Ensure this version is also identically updated in ../state-manager/Cargo.toml
 #
-sbor = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "betanet-v2-b393d4df2", features = ["serde", "indexmap"] }
-transaction = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "betanet-v2-b393d4df2" }
-radix-engine-interface = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "betanet-v2-b393d4df2", features = ["serde"]  }
-radix-engine = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "betanet-v2-b393d4df2" }
-radix-engine-constants = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "betanet-v2-b393d4df2" }
-radix-engine-stores = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "betanet-v2-b393d4df2" }
-utils = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "betanet-v2-b393d4df2" }
+sbor = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "genesis-allocation-9c475a373", features = ["serde", "indexmap"] }
```

Comment:
Now that https://github.com/radixdlt/radixdlt-scrypto/pull/790/files has merged, can we create a new tag `betanet-v2-xyz` n the scrypto repo and use that before merging :)